t/01_ps.t

```diff
@@ -1,18 +1,17 @@
 use strict;
 use warnings;
+use FindBin;
 use File::Spec;    # try to keep pathnames neutral
 use Test::More 0.96;
 
-BEGIN { use_ok("HTML::FormatPS"); }
+use lib "$FindBin::Bin/lib";
```

Comment:
If you keep your working directory as the repository root (as the installer does), then you can not only `make test`, but `prove -lr t`, etc etc and everything will DTRT.

I'm not a maintainer of this module, so my opinions are just that -- but I found this PR outstanding and was interested in getting it moving again so it could be shipped along with the other issue I filed for this distribution.